### DISCUSSION TITLE
How can I make a node with an iFrame inside clickable?

### DISCUSSION THREAD

**Discussion by CodyBontecou** [bcakmakoglu/vue-flow#1381] [Q&A] [answered]
Right now, it's defaulting to the drag state. I've tried the `nodrag` class, but wasn't getting results. Is there a recommended method to allowing the content within the node to be clickable?

**Comment by bcakmakoglu** [comment, marked as answer]
Works perfectly fine for me using `nodrag`.

[Here's a sandbox.](https://codesandbox.io/p/devbox/great-morning-hfgg27?file=%2Fsrc%2FCustomNode.vue%3A11%2C21&workspaceId=b0b5eb08-9090-4c77-9dc6-14c25ecc472e)

**Reply by CodyBontecou** [reply]
Thank you. Not sure what I was doing wrong...
